types/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "dtslint/dtslint.json",
+    "rules": {
+        "no-redundant-jsdoc": false,
+        "no-redundant-jsdoc-2": true,
+        "semicolon": false,
+        "whitespace": false
+    }
+}
```

Comment:
indent in here seems a bit off, usually it's 2 spaces with a newline at the end of the file